.buildkite/pipelines/es_snapshots/verify.yml

```diff
@@ -29,6 +29,7 @@ steps:
     agents:
       queue: ci-group-6
     depends_on: build
+    timeout_in_minutes: 150
```

Comment:
are you sure we need to change that file?

Reply:
seems that bootstrap went wrong, I'll revert it